CheckersTests/Tests.cs

```diff
@@ -0,0 +1,11 @@
+namespace CheckersTests
+{
+    public class Tests
```

Comment:
Name Class[Tests]?

Reply:
yes. it was 'xUnitTest1'. Now 'Tests'

Reply:
[.Tests](https://github.com/TopAcademy2022/translator/tree/main/Tests/translator.Tests)